app/grandchallenge/workstation_configs/serializers.py

```diff
@@ -84,4 +86,13 @@ class Meta:
             "show_flip_tool",
             "show_window_level_tool",
             "show_reset_tool",
+            "enable_contrast_enhancement",
```

Comment:
One last change - you don't need to include `enable_contrast_enhancement` now that `enabled_preprocessors` is there, this saves duplicate information in the API.